src/DotNetEnv/Parser.cs

```diff
@@ -10,22 +14,16 @@ private static bool IsComment(string line)
         private static string RemoveInlineComment(string line)
         {
             int pos = line.IndexOf('#');
-            if (pos == -1)
-                return line;
-
-            return line.Substring(0, pos);
+            return pos >= 0 ? line.Substring(0, pos) : line;
```

Comment:
This is just for simplicity / aesthetic consistency with the new `RemoveExportKeyword` implementation